cmd/ocsp-updater/main.go

```diff
@@ -129,11 +140,63 @@ func newUpdater(
 		})
 	}
 
-	updater.ocspMinTimeToExpiry = config.OCSPMinTimeToExpiry.Duration
+	if config.AkamaiBaseURL != "" {
```

Comment:
I'm inclined to tell future users to write a patch. I would like to hear how @jsha feels.